main.py

```diff
@@ -67,7 +67,8 @@ class HomePageRedirectHandler(base.BaseHandler):
     "Check whether user is logged in or logged out and perform redirects on '/'"
     def get(self):
         if self.user_id and user_services.has_fully_registered(self.user_id):
-            self.redirect(feconf.MY_EXPLORATIONS_URL)
+            #MY_EXPLORATIONS_URL
```

Comment:
Comments have a space between the hash and the first character. I'm assuming that because this is a temp item, so you'll have to remove this.